openedx/core/djangoapps/programs/tests/mixins.py

```diff
@@ -33,7 +33,10 @@ def create_programs_config(self, **kwargs):
 
 
 class ProgramsDataMixin(object):
-    """Mixin mocking Programs API URLs and providing fake data for testing."""
+    """Mixin mocking Programs API URLs and providing fake data for testing.
+
+    NOTE: This mixin is DEPRECATED. Tests should create and manage their own data.
```

Comment:
@schenedx removing this is going to require retrofitting many tests that currently depend on it. Making those changes now would increase the scope of this PR significantly. I intend to remove the mixin in a separate PR.